movie-blog/src/styles/main.less

```diff
@@ -0,0 +1,188 @@
+@base__font-size: 1.4rem;
```

Comment:
Having common CSS stylesheet for your entire app may become hard to manage. Instead, you should rather divide your styles component-wise.